WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderPostListFragment.java

```diff
@@ -405,7 +405,7 @@ public void onCreate(Bundle savedInstanceState) {
                         BuildConfig.INFORMATION_ARCHITECTURE_AVAILABLE && mIsTopLevel,
                         mRecyclerView
                 ) || ReaderUtils.isDefaultTag(mCurrentTag)) && getPostListType() != ReaderPostListType.SEARCH_RESULTS) {
-                  mViewModel.applySubfilter(subfilterListItem, true);
```

Comment:
The whole `mViewModel.getCurrentSubFilter().observe` seems a bit weird. The VM informs the view layer that `currentSubFilter` has changed -> the view layer does some business level comparisons and informs the VM back that  the currentSubFilter has changed. Is there any reason why we observe this? Couldn't we simply move the business level comparisons to the VM and call `onSubfilterChanged` there?

Again something not introduced in this PR, so feel free to ignore this comment.